Setup/shield-version.txt

```diff
@@ -1,14 +1,14 @@
-#Build Dev:Build_386 on 18/08/08 #UNR#
-SHIELD_VER=8.0.0.latest SHIELD_VER=Dev:Build_386
+#Build Dev:Build_387 on 18/08/09 #UNR#
```

Comment:
#UNR# Flag force restart of system. This mean stop all (no user surfing) and start... Please check if you really need it.